app/views/alaveteli_pro/general/_log_in_bar_links.html.erb

```diff
@@ -1,5 +1,3 @@
-<li><%= link_to _("My dashboard"), alaveteli_pro_dashboard_path %></li>
-<li><%= link_to _("My requests"), alaveteli_pro_info_requests_path %></li>
-<li><%= link_to _("My profile"), show_user_profile_path(:url_name => @user.url_name)
+<li><%= link_to _("My profile"), show_user_path(:url_name => @user.url_name)
```

Comment:
This swap of `show_user_profile_path` for `show_user_path` - could it be made everywhere, or is there a reason it should be different for pros? (It seems to end up a the same place?)